pkgs/top-level/all-packages.nix

```diff
@@ -1356,6 +1356,8 @@ with pkgs;
     python = python3;
   };
 
+  packwiz = callPackage ../tools/games/minecraft/packwiz { };
```

Comment:
So that when buildGoModule changes it's default to 18, it's easier to update

Sorry, I only saw this change in another PR after the first batch of changes

Reply:
Weird, it runs just fine with `nix run` from my testing.

Reply:
Whoops, wrong change.